ai/worker/docker.go

```diff
@@ -448,6 +450,21 @@ func (m *DockerManager) createContainer(ctx context.Context, pipeline string, mo
 	return rc, nil
 }
 
+func hwGPU(gpu string) string {
+	if !strings.Contains(gpu, "colocated-") {
+		return gpu
+	}
+
+	re := regexp.MustCompile(`colocated-\d+-(.*)`)
+	matches := re.FindStringSubmatch(gpu)
+	if len(matches) > 1 {
+		return matches[1]
+	}
+
+	clog.Warningf(context.Background(), "Invalid GPU name: %s", gpu)
```

Comment:
Maybe skip this log for `emulated-` gpus?

Reply:
Good idea. Udpated

Reply:
Ahh, wait, I think it's skipped for emulated, because it will return fast in the first line from the function.